src/pages/user/viewer/index.jsx

```diff
@@ -56,48 +65,72 @@ export const ViewUser = (props) => {
     }, []);
 
 
+    // Show loading page
     if(loading) {
         return (<PageLoading />)
-    } else {
+    } 
+    
+    // Show user page
+    else if(user) {
         return (
             <>
                 <DashboardHeader>
                     <DashboardTitle>
                         Bruker
                     </DashboardTitle>
                     <DashboardSubtitle>
-                        {user.lastname}, {user.firstname}
+                        {user.firstname} {user.lastname}
```

Comment:
But now we don't know what is part of the first and last name any more :O

Reply:
Added it again